README.md

```diff
@@ -43,29 +43,101 @@ it('should demonstrate this matcher`s usage', async () => {
 > Note, you can also require `'jest-axe/extend-expect'` which will call `expect.extend` for you.
 > This is especially helpful when using the jest `setupTestFrameworkScriptFile` configuration.
 
-### With React
+### Testing React
 
 ```javascript
+const React = require('react')
+const App = require('./app')
+
 const { axe, toHaveNoViolations } = require('jest-axe')
+expect.extend(toHaveNoViolations)
+
+it('should demonstrate this matcher`s usage with react', async () => {
+  const results = await axe(<App/>)

```

Comment:
This example should probably needs to be updated.

Reply:
What needs updating?

Reply:
The `axe` function does not accept a React component. It can only accept a string containing HTML or an HTML element. This example should probably use `react-dom`:

```javascript
import { render } from "react-dom";
render(<App />, document.body);
await axe(document.body);
```

Reply:
I've updated this in the README